### DISCUSSION TITLE
Continuous Disconnection Issue with Ant Media Live Stream

### DISCUSSION THREAD

**Discussion by Urieto** [ant-media/Ant-Media-Server#5596] [Q&A]
Hola guys, I need your help.

I am writing to bring to your attention
a persistent issue I have been experiencing with the Ant Media service
through Linode, specifically regarding the continuous 24-hour live
streaming on YouTube and Facebook.

Over the past few days since I initiated the live streams, I have
encountered a recurring problem where the streams disconnect unexpectedly.
These interruptions occur sporadically, with stream durations ranging from
as short as 2 minutes to approximately 30 minutes or sometimes an hour
before cutting off abruptly.

Despite diligently following the provided instructions and even enabling
the loop function, the automatic disconnection issue persists. This
situation is becoming increasingly frustrating as the uninterrupted 24-hour
live streaming on YouTube and Facebook was a primary motivation for
choosing Linode's services. Unfortunately, it is not performing as
advertised.

I kindly request your prompt assistance in resolving this matter. Your
immediate attention to this issue would be greatly appreciated. Please
contact me as soon as possible so that we can work together to find a
solution and ensure the uninterrupted live streaming experience that I
initially sought when choosing Linode.

PLEASE HELP 🙏🏻

**Comment by USAMAWIZARD** [comment]
I am sorry you are facing this issue Please write your problem in the following structure. Please also attach the logs 

Publishing protocol: WebRTC/RTMP/RTSP
Playing protocol: WebRTC/RTMP/HLS
Ant Media Version: 

Steps to reproduce
1.
2.
3.

Expected behavior
Put as much detail here as possible

Actual behavior
Put as much detail here as possible

**Comment by yashtandon113** [comment]
@Urieto 
As @USAMAWIZARD mention, please share the reproduce scenario and if possible, share the server logs as well. You can collect the logs from **/usr/local/antmedia/log** directory via SSH into the server.

**Comment by mekya** [comment]
I think this issue is about this one https://github.com/ant-media/Ant-Media-Server/issues/4468
